csharp-architectural-patterns/ModularMonolithPatternInCSharp/Order/Services/InventoryRabbitMqClient.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Inventory.Models;
+using Microsoft.Extensions.Configuration;
+using RabbitMQ.Client;
+using System.Text;
+using System.Text.Json;
+
+namespace Order.Services;
+
+public class InventoryRabbitMqClient(IConfiguration configuration) : IInventoryRabbitMqClient
+{
+    private readonly IConfiguration _configuration = configuration;
+
+    public void UpdateQuantity(UpdateQuantityDto updateQuantityDto)
+    {
+        var connectionFactory = new ConnectionFactory
```

Comment:
Same as for the consumer. We open connections and channels, but they are meant to be "long-lived". So we should re-use the instances as much as possible. 

Also the code looks very much the same to setup the connection itself. Can be shared :-)